iree/compiler/Conversion/HLOToHLO/test/f32Tof16.mlir

```diff
@@ -0,0 +1,38 @@
+// RUN: iree-opt -split-input-file -iree-convert-f32-to-f16 %s | IreeFileCheck %s
+
+//       CHECK: flow.variable {{.*}} : tensor<512xf16>
+// CHECK-LABEL: func @serving_default() -> tensor<512xf16>
+//  CHECK-NEXT: %{{.*}} = flow.variable.address @"__iree_flow_bert/embeddings/FakeLayerNorm/beta" : !iree.ptr<tensor<512xf16>>
+//  CHECK-NEXT: %{{.*}} = flow.variable.load.indirect %{{.*}} : !iree.ptr<tensor<512xf16>> -> tensor<512xf16>
+//  CHECK-NEXT: return %{{.*}} : tensor<512xf16>
+module attributes {tf.versions = {bad_consumers = [], min_consumer = 0 : i32, producer = 405 : i32}} {
```

Comment:
please make sure that the tes tcase is appropriately reduced. tf.versions isn't needed, and the flow.variable doesn't need to be so big.

Reply:
Good point, I simplified it.